### PR TITLE
Create 0x Mesh page

### PR DESCRIPTION
**Pull request by pauliusef**
PR creates new page `/mesh` for 0x Mesh product.

### CODE REVIEW

File: ts/pages/mesh.tsx [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+
+const descriptionCasesData = [
+    {
+        title: 'Decentralized',
+        description: 'No listing process, censorship free, globally accessible, and unstoppable',
+        icon: 'networkedLiquidity',
+    },
+    {
+        title: 'Limit orders, for more control',
+        description: 'Create markets with more granular control over pricing than AMMs like uniswap.',
```
Comment (mintcloud) [inline ts/pages/mesh.tsx]: please remove "like Uniswap".

"Create markets with more granular control over pricing than AMMs."

File: ts/utils/constants.ts [hunk 2]
```diff
@@ -135,6 +135,9 @@ export const constants = {
     URL_GITHUB_ORG: 'https://github.com/0xProject',
     URL_GITHUB_WIKI: 'https://github.com/0xProject/wiki',
     URL_FORUM,
+    URL_MESH_GUIDE_BROWSER: 'https://0x-org.gitbook.io/mesh/getting-started/rpc_api',
```
Comment (mintcloud) [inline ts/utils/constants.ts]: Please change with 

https://0x-org.gitbook.io/mesh/getting-started/browser

File: ts/utils/constants.ts [hunk 3]
```diff
@@ -135,6 +135,9 @@ export const constants = {
...
+    URL_MESH_GUIDE_NODE: 'https://0x-org.gitbook.io/mesh/getting-started/reference',
```
Comment (mintcloud) [inline ts/utils/constants.ts]: Please change with https://0x-org.gitbook.io/mesh/getting-started/deployment

File: ts/pages/mesh/liquidity_market.tsx [hunk 4]
```diff
@@ -0,0 +1,419 @@
...
+        return highestBidSlippage.minus(lowestAskSlippage).abs();
+    };
+
+    const handleSelectItem = (index: number) => {
+        setSelectedMarketIdx(index);
+    };
+
+    return (
+        <>
+            <LiquidityMarketSelect items={markets} selectedItem={selectedMarketIdx} onSelectItem={handleSelectItem} />
```
Comment (mintcloud) [inline ts/pages/mesh/liquidity_market.tsx:258]: not sure whether this is the place nor what the size specs is, but I'd like to have the LiquidityMarketSelect text a bit smaller

**Review by mintcloud — COMMENTED** [review]
Added a couple of copy changes

**Comment by pauliusef** [comment]
Made quick copy changes based on @mintcloud comments.

File: ts/pages/mesh/entry_table.tsx [hunk 5]
```diff
@@ -0,0 +1,254 @@
...
+        usedBy: '1Inch, TokenTrove',
+    },
+    {
+        title: '0x API endpoints',
+        runAs: '0x Core team',
+        aim: 'Ready to use service',
+        limitations: 'Rate limiting, token blacklist',
+        link: {
+            href: constants.URL_MESH_0X_API,
+            title: 'API specs 0xConnect',
```
Comment (mintcloud) [inline ts/pages/mesh/entry_table.tsx]: Please change this in 
"0x API SRA specifications"

File: ts/pages/mesh/entry_table.tsx [hunk 6]
```diff
@@ -0,0 +1,254 @@
...
+        link: {
+            href: constants.URL_MESH_GUIDE_NODE,
+            title: 'Mesh Guide',
+        },
+        isOpenSource: true,
+        operationCost: '$5-10 / month (based on our own DigitalOcean instances)',
+        usedBy: '1Inch, TokenTrove',
+    },
+    {
+        title: '0x API endpoints',
```
Comment (mintcloud) [inline ts/pages/mesh/entry_table.tsx]: Please change this in "0x API"

File: ts/pages/mesh/entry_table.tsx [hunk 7]
```diff
@@ -0,0 +1,254 @@
...
+        title: 'Used By',
+        hideMobile: false,
+    },
+];
+
+const entryTableData: TableEntry[] = [
+    {
+        title: 'Mesh Browser Node',
+        runAs: 'Your users',
+        aim: 'maximum decentralization, serverless setup',
```
Comment (mintcloud) [inline ts/pages/mesh/entry_table.tsx]: Please use "M" capital

Maximum

File: ts/pages/mesh/entry_table.tsx [hunk 8]
```diff
@@ -0,0 +1,254 @@
...
+        limitations: 'Warm-up time order permanence',
+        link: {
+            href: constants.URL_MESH_GUIDE_BROWSER,
+            title: 'Mesh Guide',
```
Comment (mintcloud) [inline ts/pages/mesh/entry_table.tsx]: Please change in "Mesh Browser Guide"

File: ts/pages/mesh/entry_table.tsx [hunk 9]
```diff
@@ -0,0 +1,254 @@
...
+        title: 'Mesh Node',
+        runAs: 'You',
+        aim: 'Node service and lean infra setup',
+        limitations: 'None',
+        link: {
+            href: constants.URL_MESH_GUIDE_NODE,
+            title: 'Mesh Guide',
+        },
+        isOpenSource: true,
+        operationCost: '$5-10 / month (based on our own DigitalOcean instances)',
```
Comment (mintcloud) [inline ts/pages/mesh/entry_table.tsx]: please remove "our own"

File: ts/components/dropdowns/dropdown_products.tsx [hunk 10]
```diff
@@ -16,6 +16,11 @@ const navData = [
         description: 'Embed simple token purchasing in any app or website',
         url: WebsitePaths.Instant,
     },
+    {
+        title: '0x Mesh',
+        description: 'Access token markets, permissionlessly',
```
Comment (mintcloud) [inline ts/components/dropdowns/dropdown_products.tsx:21]: for some reason, it renders the title with a capital "P". Any way we can change that?

**Review by mintcloud — COMMENTED** [review]
Adding extra comments on copy

File: ts/pages/mesh/liquidity_market_select.tsx [hunk 11]
```diff
@@ -0,0 +1,142 @@
...
+    size?: 'default' | 'small';
+}
+
+const Label = styled.span<LabelProps>`
+    font-size: ${props => (props.size === 'default' ? '26px' : '18px')};
+    line-height: ${props => (props.size === 'default' ? '32px' : '25px')};
+    opacity: ${props => (props.isMuted ? 0.7 : 1)};
+
+    @media (min-width: 768px) {
+        font-size: ${props => (props.size === 'default' ? '34px' : '20px')};
```
Comment (mintcloud) [inline ts/pages/mesh/liquidity_market_select.tsx]: suggest to change this to 28px or 30px - thanks!

Comment (pauliusef) [reply ts/pages/mesh/liquidity_market_select.tsx]: Changed the size of `default` label to 28px (line height 38px). Font size for mobile layout remains the same (26px).

**Comment by pauliusef** [comment]
Closing down this PR as the code was merged by #321

**Pull request closed without merging**